backend/app/tests/markers.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pytest import mark
+
+
+# Markers need to be registered in conftest.py and added as a return value
+# in the get_custom_markers() function.
+
+nondestructive = mark.nondestructive
+parametrize = mark.parametrize
+xfail = mark.xfail
+
+slow = mark.slow
```

Comment:
Also, this markers.py file is a bit "legacy" and was for "syntactic sugar". This file is not necessary as we register markers in `conftest.py`.

Reply:
deleted